### ISSUE TITLE
Always redirects to home page

### ISSUE THREAD

**Issue by brum1975** [markheumueller/moodle-report_deviceanalytics#10]
The default behaviour of Moodle is that if a page is requested which requires a login to access then the user is directed to the login page and once successful is presented with the page they requested. 

This plugin always directs to the homepage page, ignoring the request.

e.g. if a user is not logged in and clicks on a link such as yourmoodle.com/course/view.php?id=3 after login they should go to the course and not the home page

**Comment by brum1975** [comment]
#11 uses code from /login/index.php to redirect user to their requested page

**Comment by markheumueller** [comment]
Hello matthew,

thank you for your issue report (and your fix). I know about this problem.
It's already fixed in the new version. V2.0 will work without the [storage_helper_page.php](https://github.com/markheumueller/moodle-report_deviceanalytics/blob/master/storage_helper_page.php). 
The new version will be released as soon as I have finished my thesis. (No time right now for it)

**Comment by brum1975** [comment]
Brilliant. Good luck with your thesis!

**Comment by jrojas-garrido** [comment]
Hi @brum1975 . I undertstad you fixed this issue. But i can't understand how? Could you explain the solution  to me ? Thank You.

**Comment by brum1975** [comment]
Sorry,
I am off work and not able to remote to the server that the code is on.
Will grab it when I am back on Monday.

Looking at the code on Github I am guessing that I have modified the window.
location.replace("<?php echo $CFG->wwwroot; ?>"); in storage_helper_page.php

I suspect there is some code in the moodle/login folder that grabs the
initial url that I have used. Will know for definite when back at work.
